.gitignore

```diff
@@ -1,5 +1,5 @@
-/node_modules
+node_modules/
 .eslintrc.json
 .prettierrc
 postcss.config.js
-tailwind.config.js
```

Comment:
Can you add `.cache/` here also? And also the `dist/` folder? I think the images you upload to netlify is on the public folder then when the server compiles your code, they generate a `dist` folder. So I think it isn't necessary to add it?